src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random rnd = new Random();
+        switch (rnd.nextInt(100) % 7) {
+            case 0:
+                return "red";
```

Comment:
let's move all colors to the array of Strings or to the enum) it will look more beautifull